src/components/combobox/combobox.e2e.ts

```diff
@@ -571,6 +573,39 @@ describe("calcite-combobox", () => {
       expect(visible).toBe(true);
     });
 
+    it("when the combobox is focused & closed, Page up/down (fn arrow up/down) scrolls up and down the page", async () => {
+      const scrollablePageSizeInPx = 2400;
+      await page.addStyleTag({
+        content: `body {
+            height: ${scrollablePageSizeInPx}px;
+            width: ${scrollablePageSizeInPx}px;
+          }
+          html, body {
+            margin: 0;
+            padding: 0;
+          }
+      `
+      });
+      const combobox = await page.find("calcite-combobox");
+      await combobox.callMethod(`setFocus`);
+      const popper = await page.find("#myCombobox >>> .popper-container--active");
+      expect(popper).toBeNull();
+      expect(await page.evaluate(() => window.scrollY)).toEqual(0);
+
+      await page.keyboard.press("PageDown");
+      await page.waitForChanges();
+      expect(await page.evaluate(() => window.scrollY)).toBeTruthy();
+
+      const previousScrollPosition = await page.evaluate(() => window.scrollY);
+      await page.keyboard.press("PageUp");
+      await page.waitForChanges();
+      expect(
+        await page.evaluate((previousScrollPosition) => {
+          return window.scrollY < previousScrollPosition;
```

Comment:
Can you add a comment or store this value in a variable to express what it represents?